src/components/App/App.scss

```diff
@@ -1,8 +1,7 @@
 @import '../../variables';
 
-.app {
+body {
   background-color: $background-color;
-  height: 200vh;
 }
 
 .app-header {
```

Comment:
Merging my Header branch created 1 merge conflict in this file too.   The master branch does not have the .app-header or h1 styling because we don't need it.   

To resolve this merge conflict we will want to keep the master branch option and delete the changes from this branch.